chroma_core/lib/service_config.py

```diff
@@ -518,14 +518,78 @@ def _populate_database (self, username, password):
 
         return
 
+    def _configure_selinux(self):
+        try:
+            self.try_shell(['sestatus | grep enabled'], shell=True)
+        except CommandError:
+            return
+        
+        # This is required for opening connections between
+        # nginx and rabbitmq-server
+        self.try_shell(['setsebool -P httpd_can_network_connect 1'], shell=True)
+        
+        # This is required because of bad behaviour in python's 'uuid'
+        # module (see HYD-1475)
+        self.try_shell(['setsebool -P httpd_tmp_exec 1'], shell=True)
+
+    def _configure_firewall(self):
+        if ServiceControl.create('firewalld').running:
+            for port in [80, 443]:
+                self.try_shell(['firewall-cmd', '--permanent', '--add-port={}'.format(port)])
+                self.try_shell(['firewall-cmd', '--add-port={}'.format(port)])
+
+    def set_nginx_config(self):
+        project_dir = os.path.dirname(os.path.realpath(settings.__file__))
+        conf_template = os.path.join(project_dir, 'chroma-manager.conf.template')
+
+        nginx_settings = [
+            'APP_PATH', 'REPO_PATH', 'HTTP_FRONTEND_PORT', 'HTTPS_FRONTEND_PORT',
+            'HTTP_AGENT_PORT', 'HTTP_API_PORT', 'REALTIME_PORT', 'VIEW_SERVER_PORT',
+            'SSL_PATH', 'DEVICE_AGGREGATOR_PORT'
+        ]
+
+        with open(conf_template, "r") as f:
+            config = f.read()
+            for setting in nginx_settings:
+                config = config.replace("{{%s}}" % setting, str(getattr(settings, setting)))
+
+            with open('/etc/nginx/conf.d/chroma-manager.conf', 'w') as f2:
+                f2.write(config)
+
+    def _create_fake_bundle(self):
```

Comment:
Why do we need a fake bundle?

Reply:
Because the agent is deployed in a separate repo now, but we still need to simulate it as a bundle for the moment.